build.sh

```diff
@@ -83,6 +105,47 @@ while [[ $# -gt 0 ]]; do
         BUILD_TYPE="Debug"
         ;;
 
+    --embed-icu)
+        if [ ! -d "${CHAKRACORE_DIR}/deps/icu/source/output" ]; then
+            ICU_URL="http://source.icu-project.org/repos/icu/icu/tags/release-57-1"
+            echo "----------------------------------------------------------------"
```

Comment:
Can we replace with the following:

----------------------------------------------------------------
This script will download ICU-LIB from:
http://source.icu-project.org/repos/icu/icu/tags/release-57-1
It is licensed to you by its publisher, not Microsoft.
Microsoft is not responsible for the software.
Your installation and use of ICU-LIB is subject to the publisher’s terms available here: 
http://www.unicode.org/copyright.html#License
----------------------------------------------------------------
If you don't agree, press Ctrl+C to terminate
Hit ENTER to continue (or wait 10 seconds)